maskrcnn_benchmark/data/datasets/coco.py

```diff
@@ -83,7 +83,13 @@ def __getitem__(self, idx):
 
         if anno and "segmentation" in anno[0]:
             masks = [obj["segmentation"] for obj in anno]
-            masks = SegmentationMask(masks, img.size, mode='poly')
+            # TODO: Remove before merging
```

Comment:
auxiliary modification for unit-test of the Abs2COCO wrapping and evaluation.
Should be removed before merging